README.md

```diff
@@ -37,6 +47,7 @@ Beekeeper only actions on events which are marked with a specific parameter. Thi
 |:----|:----:|:----:|:----|
 | `beekeeper.remove.unreferenced.data=true`   | Yes |  `true` or `false`       | Set this parameter to ensure Beekeeper monitors your table for orphaned data. |
 | `beekeeper.unreferenced.data.retention.period=X` | No | e.g. `P7D` or `PT3H` (based on [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601)) | Set this parameter to control the delay between schedule and deletion by Beekeeper. If this is either not set, or configured incorrectly, the default will be used. Default is 3 days. |
+| `beekeeper.hive.event.whitelist=X` | No | Comma separated list of event types to whitelist. For example, `beekeeper.hive.event.whitelist=drop_table,drop_partition`.  | Beekeeper will always process `alter_partition` and `alter_table` events. Add other event types to this list to ensure Beekeeper does not ignore them. |
```

Comment:
Not sure if this is the most flexible, perhaps this should just be overriding?
default value = `alter_partition,alter_table`
User wants to also drop_partition he needs to set:
`beekeeper.hive.event.whitelist=alter_partition,alter_table,drop_partition`
What would be more user friendly? Currently we can never ignore `alter_table` events if anyone would ever like that?
thoughts?